Intersect.Client/Entities/Entity.cs

```diff
@@ -1636,6 +1635,27 @@ public void SortStatuses()
             Status = Status.OrderByDescending(x => x.RemainingMs()).ToList();
         }
 
+        private float GetSpriteFrames()
```

Comment:
This should be a property, `SpriteFrames`.

Reply:
Also why is this a `float`?

Reply:
Shouldn't have been a float. Fixed and is now a property.